src/core/server.js

```diff
@@ -905,6 +907,9 @@ Server.prototype._feeTxUnit = function() {
  */
 
 Server.prototype._reserve = function(ownerCount) {
+  // We should be in a valid state before calling this method
+  assert(this._reserve_base && this._reserve_inc);
```

Comment:
I'm in favour of blowing up here rather than returning some invalid Amount. It seemed like the tests before showed it was both throwing errors when not connected and returning NaN invalid amounts.